tests/containers/single_container_webui.pm

```diff
@@ -3,16 +3,26 @@ use testapi;
 use utils;
 
 sub run {
+  assert_script_run(
+        "echo  \"\$(cat <<EOF
+[global]
+# change to the URL the web UI will be available under so redirection for
+# authentication works
+base_url = http://openqa_webui
+
+[auth]
+method = Fake
+EOF
+)\" > /root/openQA/container/webui/conf/openqa.ini");
```

Comment:
I suggest to use an override file and not the complete config openqa.ini


```suggestion
)\" > /root/openQA/container/webui/conf/openqa.ini.d/fake-auth.ini");
```

Reply:
But the container doesnt provide this directory. I will have to create it manually. Altrenative , is to modify the Dockerfile again. Which I would say lets do it in another PR/ticket